File: zero/Maruko.Zero/Internal/Domain/SysRoleMenu.cs
using System.ComponentModel.DataAnnotations;
using FreeSql.DataAnnotations;
using Maruko.Core.FreeSql.Internal;

namespace Maruko.Zero
{
    [Table(Name = "sys_roleMenu")]
    public class SysRoleMenu : FreeSqlEntity
    {
        [Column(Name = "roleId")]
        public long RoleId { get; set; }

        [Column(Name = "menuId")]
        public long MenuId { get; set; }

        [Column(Name = "operates", StringLength = 500)]
        public string Operates { get; set; }
    }
}

File: act/act-modules/act-modules-zero/src/main/java/com/act/modules/zero/internal/domain/SysRoleMenu.java
package com.act.modules.zero.internal.domain;

import com.act.core.domain.BaseEntity;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

@Data
@TableName("sys_rolemenu")
@SuppressWarnings("all")
public class SysRoleMenu extends BaseEntity<Long> {

    //角色Id
    @TableField("roleId")
    private Long roleId;

    //菜单Id
    @TableField("menuId")
    private Long menuId;

    //操作
    @TableField("operates")
    private String operates;
}
